evolver/serial.py

```diff
@@ -21,12 +21,19 @@ class CMD(Enum):
         RESP_SUFFIX = b'end'
 
     class Config(pydantic.BaseModel):
+        name: str = "EvolverSerialUART"
         port: str = '/dev/ttyAMA0'
         baudrate: int = 9600
         timeout: float = 1
 
+    def __init__(self, *args, port: str = '/dev/ttyAMA0', baudrate: int = 9600, timeout: float = 1, **kwargs):
+        self.port = port
+        self.baudrate = baudrate
+        self.timeout = timeout
+        super().__init__(*args, **kwargs)
```

Comment:
as for the comment on chemostat. In this case we wouldn't even need the init here at all since we only use it to set properties from the config

Reply:
That would break inheritance.

Reply:
BTW, I didn't see any comment on chemostat?